internal/tutils/psql.go

```diff
@@ -124,9 +129,9 @@ func (p *PSQLContainer) NewDB() (db.DB, error) {
 		"--user", user,
 	)
 
-	_, err = cmd.Output()
+	out, err := cmd.CombinedOutput()
```

Comment:
This was the culprint - `Output()` is missing standard error completely, so the error was actually hidden.